PSReadLine/ReadLine.vi.cs

```diff
@@ -764,20 +753,19 @@ private static int DeleteLineImpl(int lineIndex, int lineCount)
         {
             var range = _singleton._buffer.GetRange(lineIndex, lineCount);
 
-            var deleteText = _singleton._buffer.ToString(range.Offset, range.Count);
-
-            _clipboard.LinewiseRecord(deleteText);
+            _clipboard.LinewiseRecord(_singleton._buffer.ToString(range.Offset, range.Count));

```

Comment:
@springcomp The change to `DeleteLineImpl` doesn't seem to be needed. Was it an oversight?

Reply:
It was indeed. I will revert the changes.